Sources/yoga/Algorithm_AbsoluteLayout.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#pragma once
+
+#include "Event_event.h"
```

Comment:
Cocoapods does not support header/source files with same names and different paths, Using snake case to remove duplicate file names and flatten the folder structure.
- https://github.com/CocoaPods/CocoaPods/issues/8289